sdk/messaging_servicebus/src/lib.rs

```diff
@@ -1,9 +1,7 @@
 //! Azure service bus crate for the unofficial Microsoft Azure SDK for Rust. This crate is part of a collection of crates: for more information please refer to [https://github.com/azure/azure-sdk-for-rust](https://github.com/azure/azure-sdk-for-rust).
 #![recursion_limit = "128"]
 
-// #[macro_use]
-// extern crate log;
+extern crate log;
```

Comment:
This line can be removed `extern crate` declarations are not necessary post Rust 2015

Reply:
done ✅